server/graphql/schema.js

```diff
@@ -31,9 +32,9 @@ const RootQuery = new GraphQLObjectType({
     user: {
       type: UserType,
       args: { id: { type: GraphQLID } },
-      resolve(parent, args) {
+      resolve: authenticated((parent, args) => {
```

Comment:
Now you need to wrap resolver objects with `authenticated` middleware to prevent unauthenticated users from calling them